source/mongorestore.txt

```diff
@@ -39,9 +39,11 @@ into a :binary:`~bin.mongod` or :binary:`~bin.mongos` instance.
 
 .. seealso::
 
-   :binary:`~bin.mongodump` which provides the corresponding
+   :binary:`~bin.mongodump`, which provides the corresponding
    binary data export capability.
 
+You can't run ``mongorestore`` concurrently with ``mongodump``.
```

Comment:
It's not that you can't run mongorestore concurrently with mongodump, it's that we don't support running multiple mongorestores into the same collection concurrently.

Also, I would use the phrase "not supported" instead of saying "can't". Because it is possible, it just could error.